src/wp-includes/widgets.php

```diff
@@ -1971,3 +1994,16 @@ function wp_render_widget_control( $id ) {
 
 	return ob_get_clean();
 }
+
+// Needed until src/blocks/legacy-widget/index.php in @wordpress/block-library
+// is updated to use the 'wp_' functions.
+function gutenberg_find_widgets_sidebar( $widget_id ) {
```

Comment:
Can't we update this already?

Reply:
We have to wait until `@wordpress/block-library` is published as that package includes some code which uses the `gutenberg_` functions.